tools/mo/openvino/tools/mo/utils/ir_reader/extenders/strided_slice_extender.py

```diff
@@ -18,8 +18,6 @@ def extend(op: Node):
             if op.has(attr) and op[attr] != '':
                 Extender.attr_to_list(op, attr)
             else:
-                assert attr not in ['begin_mask', 'end_mask'],\
-                    '{} is not defined for the node {}'.format(attr, op.soft_get('name', op.id))
```

Comment:
I see that `begin_mask` and `end_mask` are mandatory attributes in the specification. I think some other component erases this attribute

Reply:
If there are such atributes in IR, why do we remove this check?

Reply:
Because empty attributes are not added into the model, so MO IR Reader have extender that adds such attributes as empty array.